extensions/config-yaml/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: "YAML Configuration"
+metadata:
+  keywords:
+  - "yaml"
+  - "config"
+  - "configuration"
+  categories:
+  - "config"
+  status: "stable"
```

Comment:
This PR was just rebased onto master to solve conflicts. There's a good chance lines 8 to 10 are not correct.